README.md

```diff
@@ -54,6 +55,8 @@ In the list of commands (<kbd>CTRL</kbd> + <kbd>SHIFT</kbd> + <kbd>P</kbd>), the
 
 This displays an Explorer outline revealing the node structure of the glTF file. This allows easier navigation of the scene structure.
 
+![Gltf Outline](images/GltfOutline.png)
```

Comment:
This isn't working on the GitHub webpage but it works locally in vscode... will this be a problem until its merged?

Reply:
Ah, I missed this.  You've got a case-sensitive problem here, the file is named `GltfOutline.PNG` and should be renamed to `.png`.  Then it should work.

Committing such a rename to git on a case-insensitive system may pose a challenge.  If needed, you may need to make two commits, to alter the name and then set it back.

Reply:
Resolved